Cargo.toml

```diff
@@ -31,3 +31,6 @@ members = [
 # update dependencies often).
 [profile.dev.package."*"]
 opt-level = 3
+
+[patch.crates-io]
+geojson = { git = "https://github.com/georust/geojson", branch = "mkirk/try_from" }
```

Comment:
in general I think it's a little nicer to patch things here at the top level rather than updating across all the individual crates. It's maybe a little more convenient, but more importantly, it works for transitive deps too, which you can't conveniently edit.